lib/Event/QueryCreateEvent.php

```diff
@@ -39,12 +41,14 @@ class QueryCreateEvent extends AbstractEvent
     /**
      * @param string $innerQuery
      * @param string $locale
+     * @param array $options
      * @param null|string $primarySelector
      */
-    public function __construct($innerQuery, $locale, $primarySelector = null)
+    public function __construct($innerQuery, $locale, array $options, $primarySelector = null)
```

Comment:
Optional arguments should always come after required arguments. Should `$options` be required here?